### ISSUE TITLE
evaluate issue

### ISSUE THREAD

**Issue by NieShenRuc** [drboog/Lafite#24]
Thans for your excellent work! But i have some question about calculating fid on mscoco. 
Firstly, how do you randomly sample text from coco2014 validation set? I think there are two ways, one is sampling 30k images from 40k images then sample one captions for each image,  the other is sampling 30k captions from 200k captions in coco2014 validation set. The first way will get better fid because only one caption will be sampled for each images.
Secondly, after getting 30k generated images, i am confused about whether the coco2014 train set or validation set are used for calculating fid, especially the best fid **8.12** in you paper. Using train set to calculate will get a better fid.
I am going to cite your paper, for a reasons of fairnessI, i want to understand your evaluation method totally. i would be very appreciate if you could answer my question.

**Comment by drboog** [comment]
1. You can see how the text are sampled from dataset.py. 
2. In our code, you can use calc_metric.py and set --data to be training or validation set, which is the dataset to be compared with, --test_data to be the set that the text is sampled from. The 8.12 are calculated v.s. training set, it is around 0.1 better than comparing with validation set.

**Comment by NieShenRuc** [comment]
Thanks a lot!
